src/pricecypher/datasets.py

```diff
@@ -136,6 +136,7 @@ def get_transactions(
         start_date_time=None,
         end_date_time=None,
         bc_id='all',
+        filter_scopes=[],
```

Comment:
I think the default should be None rather than a mutable type like a list https://docs.python-guide.org/writing/gotchas/

Reply:
IMO mutating function arguments is a no-go anyway. 

Having the empty list as default value simplifies the code in this case. Using `None` instead would introduce a completely unnecessary if-statement, increasing the number of execution branches and thereby the complexity of the function.

Given that the input list is only read from and not mutated (and also should never be written to in the future), having the empty list as default value is much preferred here.

Reply:
@Eline2020 do you agree with that in this specific case?

Btw thanks for pointing this out. I didn't know about this very odd behavior in Python before. However, I'd still suggest to keep it as it is now.

Reply:
Yes, I think it's okay, since the argument isn't mutated